### DISCUSSION TITLE
Issue with new Crossplane / AWS resources showing in Kubernetes/ Crossplane

### DISCUSSION THREAD

**Discussion by bradwannow** [crossplane/crossplane#4279] [Q&A] [answered]
I am looking into migrating into Crossplane and for some reason I was able to get an S3 Bucket created successfully inside AWS using this documentation: https://docs.crossplane.io/v1.11/getting-started/provider-aws/    
However, when I run "kubectl get buckets" nothing is found and I am also seeing errors in the crossplane-system/upbound-provider-aws logs repeating with variants of the following:
```
 failed to list *v1beta1.*: metricstreams.cloudwatch.aws.upbound.io is forbidden: User "system:se │
│ rviceaccount:crossplane-system:upbound-provider-aws" cannot list resource "metricstreams" in API group "cloudwatch.aws.upbound.io" at the cluster scope: RBAC: clusterrole.rbac.authorization.k8s.io  │
│ "crossplane:provider:upbound-provider-aws-*-system" not found          
```             
So I am guessing there is some issue with my configuration, but Crossplane was able to actually create the bucket within AWS so it's not a credentials issue there. Does anybody have any idea what this could be based on the steps to that guide, that error, and the Bucket resource not showing up whatsoever in Kubernetes?

**Comment by jbw976** [comment, marked as answer]
There may be more than 1 type in the cluster that is called `buckets` - in that case, Kubernetes chooses which `buckets` type you mean and it may not get it right 😂 

See the note at the bottom of https://docs.crossplane.io/v1.11/getting-started/provider-aws/#create-a-managed-resource, copied here for convenience:

> Important
This guide uses Upbound AWS provider v0.27.0. AWS Provider v0.30.0 and later requires the full CRD name bucket.s3.aws.upbound.io instead of buckets.

So try `kubectl get bucket.s3.aws.upbound.io` and see if that lists anything? 🤔

**Reply by bradwannow** [reply]
I am using Upbound v0.27.0 so didn't think that was the issue, but do see it with `kubectl get bucket.s3.aws.upbound.io` so I guess that is the issue, maybe due to the reason the other answer stated with the bucket resource from the lightsail group.

**Comment by jeanduplessis** [comment]
This issue came in with the bucket resource that was introduced from the lightsail.aws.upbound.io group. 
If you switch to using the AWS [provider family](https://marketplace.upbound.io/providers/upbound/provider-family-aws/v0.37.0) and only run the packages you need, you could avoid this.
